R/add_CI.R

```diff
@@ -105,12 +107,11 @@ add_CI.ggsurvfit <- function(gg,
     gg <- gg +
       ggplot2::geom_ribbon(ggplot2::aes(ymin = est.lower, 
                                         ymax = est.upper, 
-                                        fill = "transparent", 
-                                        colour = strata),
-                           alpha = alpha, 
+                                        fill = strata, 
+                                        colour = strata,
+                                        alpha = alpha),
                            outline.type = "both",
                            linetype = linetype, 
-                           show.legend = FALSE,
                            na.rm = TRUE)
   }
   
```

Comment:
Can you put back the default where the CI does not color the legend surrounding the step function? This is not useful for clinical reporting. When no CI is requested we see a grey background surrounding the step function in the legend. Can this be white? It will be difficult to see when printed out.

Reply:
I removed the grey background in the legend in `visr.R`. Do we also want to get rid of the coloured background that we get when adding CIs?

Reply:
Yes I think that makes sense.

Reply:
Now no bg should ever be displayed.